optuna/samplers/_tpe/__init__.py

```diff
@@ -0,0 +1 @@
+from optuna.samplers._tpe.sampler import TPESampler  # NOQA
```

Comment:
As it's only used from `samplers/__init__.py`, maybe it's simpler to specify the full path there instead of introducing another layer of indirection? Maybe outside the scope of this PR.

Reply:
Thank you for pointing it out. I removed it and used `optuna.samplers._tpe.sampler.TPESampler` in `samplers/__init__.py`.
Commit: https://github.com/optuna/optuna/pull/1362/commits/7122120536644dc6c66a89f8796440b0e4cb8d09